includes/documents/class-wcpdf-invoice.php

```diff
@@ -53,29 +53,32 @@ public function storing_settings_enabled() {
 	 */
 	public function get_title() {
 		// override/not using $this->title to allow for language switching!
-		return apply_filters( 'wpo_wcpdf_document_title', __( 'Invoice', 'woocommerce-pdf-invoices-packing-slips' ), $this );
+		$title = __( 'Invoice', 'woocommerce-pdf-invoices-packing-slips' );
+		return apply_filters_deprecated( "wpo_wcpdf_{$this->slug}_title", array( $this->title, $this ), '3.8.7', 'wpo_wcpdf_document_title' );
```

Comment:
Is there any difference which one comes first?

Reply:
Fixed.